### ISSUE TITLE
Make cmd tutorial for pkg template setup easier to find

### ISSUE THREAD

**Issue by jamesmbaazam** [epiverse-trace/packagetemplate#21]
The [tutorial](https://github.com/epiverse-trace/packagetemplate/wiki/Command-line-instructions-to-set-up-a-package-repo) that @pratikunterwegs kindly wrote to set up the package template using the terminal is very useful but almost invisible and hard to find. Maybe, it might help to add a link in the README for visibility.

**Comment by Bisaloo** [comment]
My main blocker here is that I felt a tension between having the README as a template for the packages vs having a README talking about packagetemplate itself.

I've considered this a bit more and my conclusion is there is no probably problem to talk about packagetemplate in `README.md` while keeping `README.Rmd` as it is now. The `README.md` will be automatically overwritten as soon as the child repo gets its push first.

The only small downside is that `README.md` in packagetemplate can no longer serve as a preview of what `README.Rmd` contains. But that's maybe not a huge issue.

What do you think?

**Comment by Bisaloo** [comment]
Yes, I see the question more as: is the documentation in the wiki discoverable enough or should it be made more visible on the README. Beyond the specifics of what the documentation contains.

**Comment by pratikunterwegs** [comment]
I suppose having this information in the Readme could help external devs who like the setup of Epiverse packages and want to copy it but are not that familiar with using GH templates. Not sure how applicable that is at the moment.

**Comment by Bisaloo** [comment]
It doesn't seem that there is a great solution for this beyond what we have currently: having this piece of docs living in the wiki.

Because the packagetemplate seems to be mostly used internally and our team is aware of how to use it, I'm going to close this issue.
